### ISSUE TITLE
Code Block Unwraps on Reading Mode

### ISSUE THREAD

**Issue by rgvillanueva28** [mugiwara85/CodeblockCustomizer#105]
On reading mode, the code block unwraps even when the unwrap code block lines settings is disabled.

As an example, the image below shows the unwrapped lines in reading mode:
![image](https://github.com/user-attachments/assets/072dd6b9-f257-467b-9403-291ff67e49b5)
The image below is the output on editing mode:
![image](https://github.com/user-attachments/assets/070fbf30-0a49-4ff0-8b7a-52b333faab46)
Here is an image on reading mode with the plugin disabled:
![image](https://github.com/user-attachments/assets/62d4a107-920f-4dad-b90b-8732d0fadaa6)

### Code to reproduce:
````
```http title:request hl:5
POST /webtools/control/createUserLogin HTTP/1.1
Host: ofbiz:8443
Cookie: JSESSIONID=76A7BCC53C2B9BF49B21F99D2D8CA5D5.jvm1; OFBiz.Visitor=10000

enabled=&partyId=&userLoginId=test&currentPassword=password&currentPasswordVerify=password&passwordHint=&requirePasswordChange=N&externalAuthId=&securityQuestion=&securityAnswer=
```
````
Removing `title:request hl:5` still unwraps the code block.

**Comment by mugiwara85** [comment]
I will look at this later today

**Comment by mugiwara85** [comment]
It should be fix now. It will be released in a few days. If you want to test in the meantime just take the main.js and the styles.css files and overwrite yours wit hit (create a backup before)

**Comment by mugiwara85** [comment]
Fixed in version 1.2.8
